Admin endpoint docs (#151)
envoyproxy/xds-relay: README.md

This is a gif of a tmux session demonstrating this example:

![demo](example/xds-relay-demo.gif)

## Docs

> For more details on how to use the xDS Aggregator, see documentation [here](docs)

Reviewer: ```suggestion
For more details on how to use and observe xd-relay, see documentation [here](docs)
```
